src/api.js

```diff
@@ -23,14 +23,15 @@ export const getApplicationSchema = (
     `/api/${application}/${apiVersion}${url || `/user-config/${resourceType}`}`
   );
 
-export const saveValues = async (
-  application,
-  values,
-  apiVersion = 'v1',
-  resourceType = '',
-  url
-) =>
+export const getNotificationsSchema = (apiVersion = 'v1') =>
+  instance.get(
+    `/api/notifications/${apiVersion}/user-config/notification-event-type-preference`
```

Comment:
We should update the javascript clients to include notifications client as well.

Reply:
https://issues.redhat.com/browse/RHCLOUD-27927